Gemfile.base

```diff
@@ -93,7 +93,7 @@ end
 
 gem 'nokogiri', '>= 1.6.8'
 gem 'dalli', '~> 2.7'
-gem 'secure_headers', '~> 2.2'
+gem 'secure_headers', '~> 3.9'
```

Comment:
Why not directly moving to the last version 6.x ?

Btw rails 6 has it integrated already https://edgeguides.rubyonrails.org/security.html#content-security-policy
maybe add some notes somewhere for potential upgrade

Reply:
It will be a little bit harde to move to 6.. maybe we can do it after

Reply:
Just add a Note

Reply:
👍

Reply:
we upgraded to 6